modules/460-log-shipper/templates/daemonset.yaml

```diff
@@ -1,18 +1,18 @@
 {{- if .Values.logShipper.internal.activated }}
-{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
-{{- if eq .Values.logShipper.resourcesRequests.mode "VPA" }}
+  {{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+    {{- if eq .Values.logShipper.resourcesRequests.mode "VPA" }}
```

Comment:
Maybe `and (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") (.Values.logShipper.resourcesRequests.mode "VPA")`

I was confused by two `if` and I started looking for closing `end` but we have two closing `end` each other

Reply:
I left this code as it was previously with only an indent changing.
Speaking about double ifs, I assume this piece of code was copied from somewhere.